src/decimal.rs

```diff
@@ -548,7 +557,7 @@ impl Decimal {
     }
 
     #[must_use]
-    pub(crate) const fn from_parts_raw(lo: u32, mid: u32, hi: u32, flags: u32) -> Decimal {
+    pub const fn from_parts_raw(lo: u32, mid: u32, hi: u32, flags: u32) -> Decimal {
```

Comment:
Reverting this line:
```suggestion
    pub(crate) const fn from_parts_raw(lo: u32, mid: u32, hi: u32, flags: u32) -> Decimal {
```